kviz/dense.py

```diff
@@ -357,8 +434,11 @@ def render(self, X=None, filename='graph', duration=1000):
         vmin = min([X[:, 0].min(), X[:, 1].min()])
         vmax = max([X[:, 0].max(), X[:, 1].max()])
         norm = Normalize(vmin=vmin - 1, vmax=vmax + 1)
-        gcmap = plt.cm.Greens
-        bcmap = plt.cm.Blues
+        # gcmap = plt.cm.Greens
+        # bcmap = plt.cm.Blues
```

Comment:
please remove commented out code

Reply:
I will move them to new test cases soon. 
For now, unfortunately I cannot run tox on my machine due to it always failing to build its virtual environment. I guess it is because I am now in China and the mirror website I use does not have the right version of tf-nightly.
The error in the previous test says "E226 missing whitespace around arithmetic operator". It seems a problem of format and can be easily changed. 
Can you tell me how to run tox tests on Github? I am new to this feature and thinks it will be of great help because my local machine has difficulty with tox...
Thank you!